homeassistant/components/zabbix/__init__.py

```diff
@@ -60,6 +63,7 @@
                 vol.Optional(CONF_PASSWORD): cv.string,
                 vol.Optional(CONF_PATH, default=DEFAULT_PATH): cv.string,
                 vol.Optional(CONF_SSL, default=DEFAULT_SSL): cv.boolean,
+                vol.Optional(CONF_SENDER_PORT, default=DEFAULT_SENDER_PORT): cv.port,
```

Comment:
For integrations that connect to devices or services, we no longer accept new YAML configurations or changes.

More information about this can be found in Architecture Decision Record:

- ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

To prevent this PR from being blocked, I recommend removing the yaml port options and using the default port instead.

```suggestion
```

Reply:
Understood. I've incorporated your suggestions.